Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -325,6 +329,14 @@ - (SnapshotVersion)parseVersion:(NSNumber *_Nullable)version {
   return Version(version.longLongValue);
 }
 
+- (absl::optional<BloomFilter>)parseBloomFilter:(NSDictionary *_Nullable)bloomFilterProto {
```

Comment:
nit: Make `parseBloomFilter()` a free function defined in the anonymous namespace near the top of this file. As far as I can tell, it likely doesn't need to be a member function in the class; however, if I'm wrong then feel free to leave it as a member function.